examples/.interop/next-prisma-starter/package.json

```diff
@@ -14,7 +14,7 @@
     "prebuild": "run-s migrate generate",
     "build": "next build",
     "start": "next start",
-    "lint": "eslint --ext \".js,.ts,.tsx\" --report-unused-disable-directives --report-unused-disable-directives src",
+    "lint": "eslint --cache --ext \".js,.ts,.tsx\" --report-unused-disable-directives --report-unused-disable-directives src",
```

Comment:
the lint script is cached through turbo, so running pnpm lint in root only lints changed packages - isn't that enough? do we also need to cache the individual scripts?

Reply:
Turbo allows us to cache per package, and this allows us to cache per file: running eslint on a package multiple times (without a turbo cache hit) will take a lot less time, as it will only run on changed files instead of on each files every time.

Reply:
yea i get that - just seems a bit overkill, i rarely run a single lint locally xddd but lets go - more caching is always nice